template-compose/buildSrc/src/main/java/BuildTypes.kt

```diff
@@ -0,0 +1,4 @@
+object BuildTypes {
```

Comment:
I personally prefer the singular form for singletons, which is consistent with the naming convention of any regular class, sealed class, or enums. From the consumer's POV, the API would seem consistent too. My motivation is from [this](https://android.googlesource.com/platform/frameworks/support/+/androidx-main/compose/docs/compose-api-guidelines.md#kotlin-style-singletons_constants_sealed-class-and-enum-class-values-why) doc where they recommend using PascalCase for the constants as well, but we can keep using uppercase in this case. 💡

Reply:
@AVI5HEK I agree to use singular for sealed classes, or enums. However, this is an object class, and it's closer to what we defined with AppColors or AppDimensions in Compose template https://github.com/nimblehq/android-templates/tree/develop/template-compose/app/src/main/java/co/nimblehq/template/compose/ui/theme.